package.json

```diff
@@ -64,7 +69,13 @@
   },
   "husky": {
     "hooks": {
+      "commit-msg": "yarn sfdx-husky-commit-msg",
```

Comment:
Should this be `npm run` since the extensions don't use Yarn? And it doesn't look like `sfdx-husky-commit-msg` is a script in this package (or I might be misunderstanding how this runs).

Reply:
`sfdx-husky-commit-msg` comes from the new import '@salesforce/dev-scripts'. And I will change it to npm run, thanks for the callout :)